dev-learning/Startup.cs

```diff
@@ -21,10 +21,17 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddDbContextPool<UserContext>(
+            services.AddDbContextPool<MyDbContext>(
                options => options.UseMySql(Configuration.GetConnectionString("dbConfig")
             ));
             services.AddMvc();
+            services
+                .AddControllers()
+                .AddNewtonsoftJson(options =>
+                  {
+                     options.SerializerSettings.ReferenceLoopHandling =
+                     Newtonsoft.Json.ReferenceLoopHandling.Ignore;
+                });
```

Comment:
What do these changes do? 🤔

Reply:
This is important for having foreign keys.